docs/development/core/server/kibana-plugin-core-server.pluginconfigdescriptor.deprecations.md

```diff
@@ -4,7 +4,7 @@
 
 ## PluginConfigDescriptor.deprecations property
 
-Provider for the [ConfigDeprecation](./kibana-plugin-core-server.configdeprecation.md) to apply to the plugin configuration.
+Provider for the  to apply to the plugin configuration.
```

Comment:
Links to types moved in the `logging` and `config` packages do no longer work. Not sure there is anything we can do unfortunately.